test/libponyc-run/ffi-libponyc-standalone/main.pony

```diff
@@ -15,7 +15,7 @@ type TokenId is I32
 actor Main
   new create(env: Env) =>
     try
-      ifdef linux or osx or windows then
+      ifdef not openbsd or dragonfly then
```

Comment:
```suggestion
      ifdef not openbsd and not dragonfly then
```
all should be the above.

Reply:
I did indeed miss the parentheses. The one place where pony is happily accepting multiple operators chained without parens is `not` and I ran into it...